surveys/2022-annual-survey/questions.md

```diff
@@ -364,60 +328,96 @@ Activities:
 
 Choices:
 
-- I've *often* felt welcome
-- I *sometimes* feel welcome
-- I *never* feel welcome
+- I feel welcome
+- I feel neutral
+- I feel unwelcome
```

Comment:
I wonder if folks who *sometimes* feel unwelcome will respond differently than folks who *often* feel unwelcome. I am afraid we miss important signal by asking things this way...

Reply:
I'm not sure the difference means much in practice. I guess that if we were prioritising between venues it would, but I don't think we'd do that. Also the numbers are so small that for understanding the results it's easier to bucket these response together anyway